src/UI/Art/SpriteArtist/UnitSpriteSet.java

```diff
@@ -15,12 +15,12 @@
 
   Sprite sprites[] = new Sprite[6];
 
-  public final int ACTION_IDLE = 0;
-  public final int ACTION_MOVENORTH = 1;
-  public final int ACTION_MOVEEAST = 2;
-  public final int ACTION_MOVESOUTH = 3;
-  public final int ACTION_MOVEWEST = 4;
-  public final int ACTION_DIE = 5;
+  public static final int ACTION_IDLE = 0;
+  public static final int ACTION_MOVENORTH = 1;
+  public static final int ACTION_MOVEEAST = 2;
+  public static final int ACTION_MOVESOUTH = 3;
+  public static final int ACTION_MOVEWEST = 4;
+  public static final int ACTION_DIE = 5;
```

Comment:
...This'll be a little ugly to merge. Oops.
D'you want to muddle through that, or should I?

Reply:
I can merge.